src/styles/blocks/testimonials.scss

```diff
@@ -0,0 +1,64 @@
+.testimonials {
+  &__item {
+    margin-bottom: 48px;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+
+    @include onDesktop {
+      margin-bottom: 0;
+    }
+  }
+}
+
+.testimonial {
+  &__photo-container {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    column-gap: 24px;
+
+    margin-bottom: 28px;
+
+    @include onDesktop {
+      margin-bottom: 32px;
+    }
+  }
+
+  &__photo {
+    width: 100%;
+    grid-column: 2 / 4;
+    border-radius: 50%;
+    transition: transform 0.5s;
```

Comment:
Checklist:
11. The speed of animations is the same throughout the landing page (for example, increasing when hovering or moving blocks when scrolling)

Reply:
ready